include/mbedtls/error.h

```diff
@@ -63,6 +63,7 @@
  * CTR_DBRG  4  0x0034-0x003A
  * ENTROPY   3  0x003C-0x0040   0x003D-0x003F
  * NET      11  0x0042-0x0052   0x0043-0x0045
+ * ARIA      4  0x0058-0x005E
```

Comment:
I don't know, but since ASN1 uses error codes starting from `0x0060`, the choice is between leaving a small hole before or after ARIA (in place of the larger hole that was present between NET and ASN1 for no apparent reason). I don't think either way is better or worse than the other, so I'd rather leave it unchanged unless you disagree.